variables.tf

```diff
@@ -91,3 +91,10 @@ variable "emr_core_instance_count" {
     production  = "22"
   }
 }
+
+variable "image_version" {
+  description = "Container tag values."
+  default = {
+    s3-object-tagger = "0.0.3"
```

Comment:
this shouldn't have a default with a fixed tag. Either no default, or `latest`

Reply:
This is to pin the version instead of doing it through the pipeline

Reply:
this doesn't pin it though, it's a default